dataikuapi/dss/recipe.py

```diff
@@ -76,15 +76,25 @@ def run(self, job_type="NON_RECURSIVE_FORCED_BUILD", partitions=None, wait=True,
         :return: the :class:`dataikuapi.dss.job.DSSJob` job handle corresponding to the built job
         :rtype: :class:`dataikuapi.dss.job.DSSJob`
         """
+        project = self.client.get_project(self.project_key)
+        outputs = project.get_flow().get_graph().get_successor_computables(self)
 
-        settings = self.get_settings()
-        output_refs = settings.get_flat_output_refs()
-
-        if len(output_refs) == 0:
+        if len(outputs) == 0:
             raise Exception("recipe has no outputs, can't run it")
 
-        jd = self.client.get_project(self.project_key).new_job(job_type)
-        jd.with_output(output_refs[0], partition=partitions)
+        first_output = outputs[0]
+
+        object_type_map = {
+            "COMPUTABLE_DATASET": "DATASET",
+            "COMPUTABLE_FOLDER": "MANAGED_FOLDER",
+            "COMPUTABLE_SAVED_MODEL": "SAVED_MODEL",
+            "COMPUTABLE_STREAMING_ENDPOINT": "STREAMING_ENDPOINT",
```

Comment:
So I did the test and `STREAMING_ENDPOINT` can be run, but it will produce an empty job. Instead they can be started or stopped (and we have a `dataikuapi.dss.continuousactivity.DSSContinuousActivity` for that). 

So we should:
* either fail with the generic error
* or be a little nicer and output a contextualized message saying that they should use `DSSContinuousActivity` instead

What do you think ?

Reply:
I have no strong opinion one way or the other 😅

Reply:
After discussing with @FChataigner, it seems that we should keep the ability to run a recipe with `STREAMING_ENDPOINT` output, for the case of Sync recipes where it is possible to sync dataset -> streaming endpoint. 

For the other cases, we can fail with a contextualized message

Reply:
Revert changes related to continuous edge case to keep it simple in this PR